shared/login/reset-modal/container.tsx

```diff
@@ -0,0 +1,17 @@
+import * as Container from '../../util/container'
```

Comment:
IMO let's delete this until it's integrated

Reply:
it's there to make it testable, otherwise you pretty much cant see the whole thing in action

Reply:
BUT this whole thing might be changed, there's been changes in what we want to put in here

Reply:
It's usually the integration's job to add the container and in-app entry point for this at once